chore(curriculum): add two labs to full stack/javascript/loops (#60185)
freeCodeCamp/freeCodeCamp: client/src/pages/learn/full-stack-developer/lab-chunky-monkey/index.md

---
title: Introduction to the Chunky Monkey Algorithm
block: lab-chunky-monkey
superBlock: full-stack-developer
---

## Introduction to the Chunky Monkey Algorithm

> In this lab you will practice dividing an array in smaller arrays with the technique of your choice.

Reviewer: if you leave a suggestion with your preferred intro I will add it

> In this lab you will practice dividing an array in smaller arrays with the technique of your choice.

Reviewer 2: ```suggestion
This lab will focus on dividing an array into smaller arrays.
```
